app/scripts/views/sign_in.js

```diff
@@ -95,6 +96,12 @@ define(function (require, exports, module) {
       this._formPrefill.set('password', this.getElementValue('.password'));
     },
 
+    hideSuccessAndErrorMessages: function () {
+      $(this.el).off('keypress', '.input-row');
+      BaseView.prototype.hideSuccess.apply(this, arguments);
```

Comment:
`hideSuccess` should already be available because all views inherit from the BaseView. So you don't need to call `BaseView` directly here

See:
![image](https://cloud.githubusercontent.com/assets/128755/13927141/dc13f20e-ef65-11e5-8318-a58accb20f29.png)